config/sidekiq.yml

```diff
@@ -13,3 +13,8 @@
   - ["notifications_low", 2]
   - ["notifications", 1] # TODO: this queue can be deleted in a future PR once high/medium/low are in place
   - ["default", 1]
+
+:schedule:
+  all_worker:
+    cron: '0 * * * * *'
```

Comment:
we definitely do not want to run all the feed jobs every minute 🙀

Reply:
Haha, good catch, changed this to once per minute to test it actually scheduled them, forgot to change it back 🤦